docs/standard/threading/countdownevent.md

```diff
@@ -43,4 +43,4 @@ ms.author: "ronpet"
   
 ## See also
 
-- [EventWaitHandle, CountdownEvent](../../../docs/standard/threading/eventwaithandle-autoresetevent-countdownevent-manualresetevent.md)
+- <xref:System.Threading.Semaphore?displayProperty=nameWithType>
```

Comment:
Link to semaphore, because `CountdownEvent` can be thought of as the opposite/reverse semaphore.